funding/manager.go

```diff
@@ -2788,16 +2829,16 @@ func (f *Manager) handleFundingConfirmation(
 		&fundingPoint, markedOpen, &confChannel.shortChanID,
 	)
 	if err != nil {
-		return fmt.Errorf("error setting channel state to markedOpen: %v",
-			err)
+		return fmt.Errorf("error setting channel state to "+
+			"markedOpen: %v", err)
```

Comment:
I'm curious, did the linter complain about these just because you touched some parts of the file? Or did you add those just for completeness? Either way, I think pure formatting changes should be in their own commit to not distract from the main changes.

Reply:
I think I saw the new linter warnings after I made these changes. So I did it just for completeness.
However the linter flagged this one below so I changed `%v` to `%w`: https://github.com/lightningnetwork/lnd/pull/6903/commits/be5d00bee1a5d4a50506b7881a2706d7c7626554#diff-bbccb27bb17be513ad84c77ad8e37a4d76168956f7978309aab29a0cbfdf91acR3263